pkg/virt-controller/watch/vmi.go

```diff
@@ -380,6 +380,40 @@ func (c *VMIController) updateStatus(vmi *virtv1.VirtualMachineInstance, pod *k8
 			conditionManager.RemoveCondition(vmiCopy, virtv1.VirtualMachineInstanceConditionType(k8sv1.PodReady))
 		}
 
+		var networkPatchNeeded bool
+		existingNetworks := map[string]virtv1.Network{}
```

Comment:
What is an `existingNetwork`? It sounds a little too generic and does not help me understand what this keeps. Maybe `vmiNetworksByName` would be better?